models/cuboid/src/main.rs

```diff
@@ -1,6 +1,31 @@
-fn main() -> fj::Result {
-    let mut fj = fj::Instance::new();
-    let model = cuboid::model([3., 2., 1.], &mut fj.core);
-    fj.process_model(&model)?;
+use clap::Parser;
+use fj::{Args, Instance};
+
+#[derive(Parser)]
+struct Parameters {
+    /// Size of the cuboid, as a comma-separated vector `x,y,z`
+    #[arg(long, value_parser = parse_vector_3, default_value = "1,1,1")]
```

Comment:
Please change the new defaults to match the old ones. Having different sizes for the different dimensions makes sense for a test model, as it can make it easier to recognize where a mistake is, or spot the mistake in the first place.